tcms/testruns/static/testruns/js/get.js

```diff
@@ -8,12 +8,17 @@ $(document).ready(() => {
 
     $('#status_button').on('switchChange.bootstrapSwitch', (_event, state) => {
         if (state) {
-            jsonRPC('TestRun.update', [testRunId, { 'stop_date': null }], () => { })
+            jsonRPC('TestRun.update', [testRunId, { 'stop_date': null }], () => {
+                $('.stop-date').html('-')
+            })
         } else {
             const timeZone = $('#clock').data('time-zone')
             const now = currentTimeWithTimezone(timeZone)
 
-            jsonRPC('TestRun.update', [testRunId, { 'stop_date': now }], () => { })
+            jsonRPC('TestRun.update', [testRunId, { 'stop_date': now }], testRun => {
+                const stopDate = moment(testRun.stop_date).format("DD MMM YYYY, HH:mm a")
+                $('.stop-date').html(stopDate)
```

Comment:
The format is slightly different from what Django uses, but also Django formats can vary based on settings and location.  The difference is probably not very visible so let's leave it like that for now.